src/index.html

```diff
@@ -4,7 +4,7 @@
   <meta charset="utf-8">
   <title>EvalAI</title>
   <base href="/">
-
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
```

Comment:
@harshit98 please don't use bootstrap. You have to write these component from scratch.